src/components/MainScreen/MainScreen.tsx

```diff
@@ -6,50 +6,63 @@ import Image from "../Image/Image";
 import CategorySelect from "../CategorySelect/CategorySelect";
 import AlertSnackbar from "../AlertSnackbar/AlertSnackbar";
 import { AxiosError } from "axios";
+import InputForm from "../InputForm/InputForm";
 
 const MainScreen = () => {
   const [joke, setJoke] = useState<ChuckJoke>();
   const [errorMessage, setErrorMessage] = useState<string>();
-  const [category, setCategory] = useState('')
+  const [category, setCategory] = useState("");
   const [isError, setIsError] = useState(false);
+  const [customName, setCustomName] = useState<string>();
```

Comment:
doesn't typescript mark this as error? 
you type this state as string, yet pass no string as default state, and later in onFormSubmit pass 
setCustomName(customName)
where customName has type: 
customName: string | undefined